charm/docs/tutorial/getting-started.md

```diff
@@ -75,41 +85,47 @@ This allows uploaded images to be placed on an S3 compatible server instead of
 the local filesystem.
 
 You can configure it with:
```

Comment:
I would separate these commands into separate steps since a lot is going on and the `juju config` command requires the user to set some environment variables. In fact, maybe there should also be a command block and instructions for the user to set the env vars. I have included a starting point below, although maybe you will have a better explanation of the AWS variables :)

---

Deploy the s3-integrator charm:

```bash
juju deploy s3-integrator --channel edge
```

Configure the s3-integrator charm with your AWS information, and then wait for the charm to update:
```bash
juju config s3-integrator endpoint="${AWS_ENDPOINT_URL}" bucket="${AWS_BUCKET}" path=/ region="${AWS_REGION}" s3-uri-style="${AWS_URI_STYLE}"
juju wait-for application s3-integrator --query='name=="s3-integrator" && (status=="active" || status=="blocked")'
```

Once the s3-integrator charm has been deployed, integrate the charm with NetBox:

```bash
juju integrate s3-integrator netbox
```